src/App.tsx

```diff
@@ -1,29 +1,45 @@
-import { Component } from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { FindMovie } from './components/FindMovie';
+import { getMovie } from './api';
 
-interface State {
-  movies: Movie[];
-}
+export const App: React.FC = () => {
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [newMovie, setMovie] = useState<Movie | null>(null);
+  const [movieFound, setFoundMovie] = useState(true);
 
-export class App extends Component<{}, State> {
-  state: State = {
-    movies: [],
+  const addMovieToList = (movie: Movie) => {
+    if (movie !== null && movies.every(movieItem => movieItem.imdbID !== movie.imdbID)) {
```

Comment:
```suggestion
    if (movie  && movies.every(movieItem => movieItem.imdbID !== movie.imdbID)) {
```

Reply:
For such condition, better to create a variable, for example: `isMovieAlreadyAdded`